### DISCUSSION TITLE
Posterior Distribution

### DISCUSSION THREAD

**Discussion by sv09** [google/lightweight_mmm#88] [Q&A] [answered]
Hi team,

I'm looking for some clarity on posterior distribution graphs. What is the x-axis and y-axis here? I have a general idea that these represent distribution of probabilities, but it would be helpful to get some insight in terms of how to interpret these graphs for media channels. 

![1](https://user-images.githubusercontent.com/38870742/195639236-fdcb5f02-7b5b-4b28-aebd-b8eb8083ecf4.png)

What does it mean if a channel follows a normal distribution curve vs one that follows an exponential curve? For example, how would we interpret the posterior distribution graph for channel 0 and for channel 2 in the figure above?

Thank you!

**Comment by pabloduque0** [comment, marked as answer]
Yes exactly they are probability distributions. So `x` will be the value and `y` will be the frecuency.

When predicting the values of the parameters will be sampled from these set of posterior distributions. You can also use these posterior to obtain model insights (most plots in the package do that).

Hopefully that helps but let me know if something is unclear.

**Reply by sv09** [reply]
Thank you for the clarification @pabloduque0!

**Comment by sv09** [comment]
Hi Pablo,

Following up on the explanation above - "So x will be the value and y will be the frecuency.", I was wondering what is the value of x-axis when we input channel spend amount to the model. I see that channel posterior values are obtained using media_mix_model.trace["coef_media"]. What does this method do?

**Reply by pabloduque0** [reply]
The value in the `x` axis will be the value of the coefficient, so the value that will be multiplied by your data to obtain the media contribution. Maybe the [model documentation](https://lightweight-mmm.readthedocs.io/en/latest/models.html) can shed some light for you to better understand what/how we are modelling things.
